aea/cli/check_packages.py

```diff
@@ -541,20 +551,15 @@ def run(self) -> None:
         )
 
         if missed_deps_for_imports:
-            print(f"unresolved imports: {', '.join(missed_deps_for_imports)}")
+            raise PypiDependencyNotFound(
+                f"unresolved imports: {', '.join(missed_deps_for_imports)}"
+            )
```

Comment:
We should utilise the `CustomError` class to print errors, the same way we do in the other checks